maro/cli/k8s/executors/k8s_azure_executor.py

```diff
@@ -51,11 +51,10 @@ def build_cluster_details(create_deployment: dict):
     @staticmethod
     def _validate_create_deployment(create_deployment: dict):
         optional_key_to_value = {
-            "root['master']['redis']": {'port': 6379},
+            "root['master']['redis']": {"port": 6379},
```

Comment:
Can you make redis port configable?

Reply:
it is already configurable, only when the user does not specify it, the system assigns a default value 6379 to it.